app/javascript/components/orchestration-template/orcherstration-template-form.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid } from 'patternfly-react';
+
+import { API } from '../../http_api';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import orchestrationFormSchema from './orchestration-template-form.schema';
+
+const submitNewTemplate = (values, message) => API.post('/api/orchestration_templates', values)
+  .then(() => miqRedirectBack(message, 'success', '/catalog/explorer'))
+  .catch(() => miqSparkleOff());
+
+const updateTemplate = (values, message, otId) => API.patch(`/api/orchestration_templates/${otId}`, values)
+  .then(() => miqRedirectBack(message, 'success', '/catalog/explorer'))
+  .catch(() => miqSparkleOff());
+
+const copyTemplate = (values, message, otId) => API.post(`/api/orchestration_templates/${otId}`, {
+  action: 'copy',
+  resource: values,
+}).then(() => miqRedirectBack(message, 'success', '/catalog/explorer'))
+  .catch(() => miqSparkleOff());
+
+const OrcherstrationTemplateForm = ({ otId, copy }) => {
+  const [initialValues, setinitialValues] = useState({});
+  const [submitAction, setSubmitAction] = useState();
+  const [isLoading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setSubmitAction(() => (copy ? copyTemplate : otId ? updateTemplate : submitNewTemplate));
+    if (otId) {
+      API.get(`/api/orchestration_templates/${otId}?attributes=name,description,type,ems_id,draft,content`)
+        .then((data) => {
+          setinitialValues(copy ? { ...data, name: `Copy of ${data.name}` } : data);
+          setLoading(false);
+        });
+    } else {
+      setLoading(false);
+    }
+  }, []);
+
+  const schema = orchestrationFormSchema(!!otId, copy, initialValues);
+
+  const onSubmit = ({ href: _href, id: _id, ...values }) => {
+    miqSparkleOn();
+    const successMessage = sprintf(__(`Orchestration Template %s was ${otId ? 'saved' : 'successfully created'}`), values.name);
```

Comment:
I do not think that the parser collecting the strings works. If it's based on syntactic analysis like the Ruby one, then this will not work.
You need 2 strings and no function call inside the __(...)